android/app/build.gradle

```diff
@@ -151,19 +150,19 @@ android {
 
 dependencies {
 
-    implementation 'com.facebook.soloader:soloader:0.9.0+'
```

Comment:
Not sure if this version is required for the current react-version or belongs to the newest, however it says 0.9.0+ might be that we are using the latest 🤔

Reply:
having + on the version actually tries to use the latest version available, but using + on the version is considered a bad thing because it adds some uncertainties to the build. I think this blog gives a good explanation on it : https://blog.danlew.net/2015/09/09/dont-use-dynamic-versions-for-your-dependencies/